Source/WebKit/UIProcess/Cocoa/AuxiliaryProcessProxyCocoa.mm

```diff
@@ -49,7 +49,9 @@
     if (noErr != AudioComponentFetchServerRegistrations(&registrations) || !registrations)
         return nullptr;
 
-    return WebCore::SharedBuffer::create(registrations);
+    auto audioRegistrations = adoptCF(registrations);
+    
+    return WebCore::SharedBuffer::create(audioRegistrations.get());
```

Comment:
Indeed, the AudioComponentFetchServerRegistrations() doc says the caller needs to release the registrations :(

I'd rather we fix it this way though:
return WebCore::SharedBuffer::create(adoptCF(audioRegistrations).get());

Reply:
> Indeed, the AudioComponentFetchServerRegistrations() doc says the caller needs to release the registrations :(
> 
> I'd rather we fix it this way though: return WebCore::SharedBuffer::create(adoptCF(audioRegistrations).get());

Ah, yes, good point! Fixed in latest patch.

Thanks for reviewing!